registers/trillian_client/client.go

```diff
@@ -49,25 +50,24 @@ func (t *trillianClient) Scan(logID int64, s LogScanner) error {
 		g := &trillian.GetLeavesByRangeRequest{LogId: logID, StartIndex: n, Count: CHUNK}
 		r, err := t.tc.GetLeavesByRange(ctx, g)
 		if err != nil {
-			log.Fatalf("Can't get leaf %d: %v", n, err)
+			fmt.Errorf("Can't get leaf %d: %v", n, err)
```

Comment:
lower case initials on error strings (all the other log -> error conversion in this commit too)